v2/api/kubernetesconfiguration/v1api20221101/structure.txt

```diff
@@ -0,0 +1,267 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+github.com/Azure/azure-service-operator/v2/api/kubernetesconfiguration/v1api20221101
+├── APIVersion: Enum (1 value)
+│   └── "2022-11-01"
+├── Extension: Resource
+│   ├── Spec: Object (13 properties)
+│   │   ├── AksAssignedIdentity: *Object (1 property)
+│   │   │   └── Type: *Enum (2 values)
+│   │   │       ├── "SystemAssigned"
+│   │   │       └── "UserAssigned"
+│   │   ├── AutoUpgradeMinorVersion: *bool
+│   │   ├── AzureName: string
+│   │   ├── ConfigurationSettings: map[string]string
+│   │   ├── ExtensionType: *string
+│   │   ├── Identity: *Object (1 property)
+│   │   │   └── Type: *Enum (1 value)
+│   │   │       └── "SystemAssigned"
+│   │   ├── Owner: *genruntime.ArbitraryOwnerReference
+│   │   ├── Plan: *Object (5 properties)
+│   │   │   ├── Name: *string
+│   │   │   ├── Product: *string
+│   │   │   ├── PromotionCode: *string
+│   │   │   ├── Publisher: *string
+│   │   │   └── Version: *string
+│   │   ├── ReleaseTrain: *string
+│   │   ├── Scope: *Object (2 properties)
+│   │   │   ├── Cluster: *Object (1 property)
+│   │   │   │   └── ReleaseNamespace: *string
+│   │   │   └── Namespace: *Object (1 property)
+│   │   │       └── TargetNamespace: *string
+│   │   ├── Statuses: Object (5 properties)[]
```

Comment:
Hmm, can this field actually be set? Given its name I am wondering if it's really supposed to be marked readonly?

Reply:
Checking on this

Reply:
Done!